src/imitation/algorithms/bc.py

```diff
@@ -192,7 +222,7 @@ def train(
         samples_so_far = 0
         batch_num = 0
         for epoch_num in trange(n_epochs, desc="BC epoch"):
```

Comment:
It would be very cool if you could refactor this to run for a fixed # of batches instead of a fixed # of epochs (like the GAIL training code does). The concept of an "epoch" is a bit awkward when you only have datasets of a few dozen or a few hundred time steps :(

(don't feel pressured to do it this PR, though)

Reply:
I think epochs is very natural for supervised learning on a fixed dataset. Probably save this for #79 if we want to think about unifying the algorithms' interfaces.